src/game/engine/display.h

```diff
@@ -81,6 +81,7 @@ class DisplayClass : public MapClass
 
     coord_t Pixel_To_Coord(int x, int y);
     void Set_Cursor_Shape(int16_t *list = nullptr);
+    void Clear_Cursor_Shape() { Set_Cursor_Shape(nullptr); }
```

Comment:
What does this do that calling `Set_Cursor_Shape();` doesn't do? Seems pointless, its even slightly longer to type.